contrib/localnet/README.md

```diff
@@ -76,6 +76,47 @@ which does the following docker compose command:
 # in zeta-node/contrib/localnet/orchestrator
 $ docker compose down --remove-orphans
 ```
+### Run monitoring setup
+Before starting the monitoring setup, make sure you have added the eth addresses of interest to the grafana/addresses.txt file
```

Comment:
Can you include the urls to access the grafana dashboard locally and the default UN/PW please.

Reply:
Updated now